common/models/user.js

```diff
@@ -1,3 +1,33 @@
+var mailerService = require('../../server/server-utils/emailProvider');
+
 module.exports = function(user) {
+    var User = user;
+
+
+    User.on('resetPasswordRequest', function(info){
+      var MailService = new mailerService();
+      var accessToken = info.accessToken.id;
+      var email = info.email;
+      var user_id = info.accessToken.userId;
+
+      var resetURL = 'http://localhost:8080/resetPassword/?id=' + user_id + '&token=' + accessToken;
```

Comment:
This hard-coded url seems bad. We can have it hard-coded in dev mode to this url (NODE_ENV environment variable not set to "production"), but on heroku, we need to have this come from an environment variable, and if that variable isn't defined, we should probably fail the build.